README.md

```diff
@@ -686,6 +686,113 @@ avalanche-network-runner control start \
 --blockchain-specs '[{"vm_name": "blobvm", "genesis": "/tmp/blobvm.genesis.json", "chain_config": "'$CHAIN_CONFIG_PATH'", "network_upgrade": "'$NETWORK_UPGRADE_PATH'"}]'
 ```
 
+## `network-runner` RPC server: `timestampvm` example
+
+To start the server:
+
+```bash
+avalanche-network-runner server \
+--log-level debug \
+--port=":8080" \
+--grpc-gateway-port=":8081"
+
+# make sure network-runner server is up
+curl -X POST -k http://localhost:8081/v1/ping -d ''
+```
+
+To start the cluster with custom chains:
+
+```bash
+# or download from https://github.com/ava-labs/subnet-cli/releases
+cd ${HOME}/go/src/github.com/ava-labs/subnet-cli
+go install -v .
+subnet-cli create VMID timestampvm
+# tGas3T58KzdjcJ2iKSyiYsWiqYctRXaPTqBCA11BqEkNg8kPc
+
+# download from https://github.com/ava-labs/avalanchego/releases
+# or build
+rm -rf ${HOME}/go/src/github.com/ava-labs/avalanchego/build
+cd ${HOME}/go/src/github.com/ava-labs/avalanchego
+./scripts/build.sh
+
+# or download from https://github.com/ava-labs/timestampvm/releases
+# cd ${HOME}/go/src/github.com/ava-labs/timestampvm
+# ./scripts/build.sh
+cd ${HOME}/go/src/github.com/ava-labs/timestampvm
+go build -v \
+-o ${HOME}/go/src/github.com/ava-labs/avalanchego/build/plugins/tGas3T58KzdjcJ2iKSyiYsWiqYctRXaPTqBCA11BqEkNg8kPc \
+./main
+
+# make sure binaries are built
+find ${HOME}/go/src/github.com/ava-labs/avalanchego/build
+# for example
+# .../build
+# .../build/plugins
+# .../build/plugins/tGas3T58KzdjcJ2iKSyiYsWiqYctRXaPTqBCA11BqEkNg8kPc
+# .../build/plugins/evm
+# .../build/avalanchego
+
+# generate the genesis for the custom chain
```

Comment:
Maybe add a short comment saying that the timestampvm just requires a non-empty genesis with bogus content or something like that.